Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -1178,4 +1179,59 @@
     <string name="test_api_body_other_keys">"Other key"</string>
     <!-- NOTR -->
     <string name="test_api_calculate_risk_level">"Calculate Risk Level"</string>
+
+    <string name="country_name_all">Alle Länder</string>
+    <string name="country_name_at">Österreich</string>
```

Comment:
```
<!-- XTXT: interoperability: countrylist - the user sees a list of all available countries -->
```

Reply:
As the translators see the value you can just reuse this comment for all country strings.